dev/integration_tests/web/lib/null_enabled_api.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.10
+// @dart = 2.12
```

Comment:
Are these non-opt-out ones necessary anymore?

Reply:
the integration_test/web package hasn't been migrated and I didn't really want to do that as part of this PR. I don't think the changes would be big